tests/teststub/characters.go

```diff
@@ -75,3 +71,16 @@ func (s *Characters) getCharacterEval(idx int) testeval.ActionEval {
 	}
 	return nil
 }
+
+func (s *Characters) init(characters []key.TargetID) {
+	if len(s.testChars) < len(characters) { // eval loaded config, need to populate testchars
+		for i := range s.cfg.Characters {
+			s.testChars = append(s.testChars, newCharacter(s.cfg.Characters[i]))
+		}
+	}
+	for i := range s.testChars {
+		s.testChars[i].key = characters[i]
+		s.testChars[i].stat = s.attributes.Stats
+		s.testChars[i].t = s.t
+	}
```

Comment:
do we need to reset the `model` here (not looking at how this is being used)

Reply:
not necessary as only the Key is relevant